### DISCUSSION TITLE
My Personal Profile

### DISCUSSION THREAD

**Discussion by KaweMaximo** [community/community#29740] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I'm new to the Development Area and I wanted connections/friends, to create a link for both Studies and Projects.

Who can be evaluating and giving tips what I can study to improve my knowledge in this area, I will be grateful <3.

 [My GitHub](https://github.com/KaweMaximo)

**Comment by radar-blip** [comment, marked as answer]
Your links are not working.
Try this:
```
 [My GitHub](https://github.com/KaweMaximo)
```

**Reply by KaweMaximo** [reply]
Wow, thanks buddy I hadn't seen it :P

**Reply by radar-blip** [reply]
No problem. Glad you got it working! 👍🏼

**Reply by marran2019** [reply]
How to unlock my facebook account?
My account is 2 months blocks
